packages/terra-data-grid/src/subcomponents/ColumnResizeHandle.jsx

```diff
@@ -85,28 +114,115 @@ const ColumnResizeHandle = (props) => {
     event.preventDefault();
   };
 
+  const onMouseUp = () => {
+    onResizeMouseUp();
+  };
+
+  const [isAriaLabel, setIsAriaLabel] = useState(false);
+  const [isAriaValueText, setIsAriaValueText] = useState(false);
+
+  const fitToTable = () => {
+    // Find parent table element
+    const parentTable = resizeHandleRef.current.closest('table');
+
+    // Update resize handle height to match parent table height
+    if (parentTable) {
+      resizeHandleRef.current.style.height = `${parentTable.offsetHeight}px`;
+    }
```

Comment:
You should be able to get the table height from the prop passed by the DataGrid component down to the subcomponents.

Reply:
Changed here:

Reply:
and removed `if` block here: 9569340cc2fa8ff84ac8eb88dff2325fb14cd05c